iHome/ihome.frontend/src/app/components/add-device/add-device.component.spec.ts

```diff
@@ -0,0 +1,23 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
Czy to są UNIT TESTY?
Ale fajnie xd, można dodać potem jakieś prawdziwe testy poza should create. Oczywiście jeżeli ten komponent coś robi

Reply:
Ta są unit testy w angularku